### PR TITLE
Removing duplicates in tags object

### PR DESCRIPTION
**Pull request by RoshniNaveenaS**
Tags object is now put in a set to address error of duplicates in tags:
image

### CODE REVIEW

File: test/lib/compile/csdl2openapi.test.js [hunk 1]
```diff
@@ -103,6 +103,22 @@ describe('Edge cases', function () {
...
+            this: {
+                ET: { $Kind: 'EntityType', $Key: ['key'], key
+                : {} },
+                Container: {
+                    Set: { $Type: 'this.ET', $Collection: true }
+                }
+            }
+        };
+        const openapi = lib.csdl2openapi(csdl, {});
+        assert.ok(openapi.tags, 'Tags object exists');
```
Comment (RamIndia) [inline test/lib/compile/csdl2openapi.test.js:118]: How are you providing duplicate values in this sample?

Comment (RoshniNaveenaS) [reply test/lib/compile/csdl2openapi.test.js]: made changes

File: lib/compile/csdl2openapi.js [hunk 2]
```diff
@@ -496,18 +496,19 @@ module.exports.csdl2openapi = function (
      * @return {Array} The list of tags
      */
     function getTags(container) {
-        const tags = [];
-        // all entity sets and singletons
```
Comment (jeevitha011) [inline lib/compile/csdl2openapi.js:500]: comment need not be removed

File: test/lib/compile/openapi.test.js [hunk 3]
```diff
@@ -117,6 +117,40 @@ describe('OpenAPI export', () => {
...
+            totalAmount : Decimal(10, 2);
+            product     : Association to Products;
+    }
+
+}
+    `);
+
+    const openAPI = toOpenApi(csn);
+    expect(openAPI).toBeDefined();
+    console.log(openAPI.tags);
```
Comment (jeevitha011) [inline test/lib/compile/openapi.test.js:150]: logs need to be removed

File: lib/compile/csdl2openapi.js (hunk 2 above)
Comment (RoshniNaveenaS) [reply lib/compile/csdl2openapi.js:500]: made changes

File: test/lib/compile/openapi.test.js (hunk 3 above)
Comment (RoshniNaveenaS) [reply test/lib/compile/openapi.test.js]: made changes

**Review by jeevitha011 — APPROVED** [review]
